docs/reference/errors-and-warnings/NU3028.md

```diff
@@ -15,10 +15,21 @@ f1_keywords:
 
 *NuGet 4.6.0+*
 
-<pre>The revocation function was unable to check revocation for the certificate.</pre>
+<pre>The author primary signature's timestamp found a chain building issue: The revocation function was unable to check revocation because the revocation server could not be reached. For more information, visit https://aka.ms/certificateRevocationMode</pre>
 
 ### Issue
 Certificate chain building failed for the timestamp signature. The timestamp signing certificate is untrusted, revoked, or revocation information for the certificate is unavailable.
 
 ### Solution
 Use a trusted and valid certificate. Check internet connectivity.gits
+
+### Revocation check mode
+If the machine has restricted internet access (such as a build machine in a CI/CD scenario), installing/restoring a signed nuget package will result in this warning since the revocation servers are not reachable. This is expected.
+However, in some cases, this may have unintended concequences such as the package install/restore taking longer than usual. If that happens, you can work around it by setting the `NUGET_CERT_REVOCATION_MODE` environment variable to `offline`. This will force NuGet to check the revocation status of the certificate only against the cached certificate revocation list, and NuGet will not attempt to reach revocation servers.
```

Comment:
Since nuget.exe is already published, I have to publish the docs. Please create a PR to the docs if you think we need to add more info around OSCP and CRL.